benchmarks/go/signable_starknet.go

```diff
@@ -63,7 +63,7 @@ func (o *OrderPayload) FmtDefinitionEncoding(field string) (fmtEnc []*big.Int) {
 	case "market":
 		fmtEnc = append(fmtEnc, types.StrToFelt(o.Market).Big())
 	case "side":
-		fmtEnc = append(fmtEnc, types.StrToFelt(OrderSide(o.Side).ChainId()).Big())
+		fmtEnc = append(fmtEnc, types.StrToFelt(OrderSide(o.Side).Get()).Big())
```

Comment:
what does Get do?

Reply:
Gets the order side in chain format (BUY = 1, SELL = 2)

`ChainId` was inaccurate method name.